File: Java Project/Calculator.java
public class Calculator {
    public static void main(String[] args) {
        System.out.println("Welcome to the Java Calculator!");
        int a = 10;
        int b = 5;
        System.out.println("Add: " + Operations.add(a, b));
        System.out.println("Subtract: " + Operations.subtract(a, b));
        System.out.println("Multiply: " + Operations.multiply(a, b));
        System.out.println("Divide: " + Operations.divide(a, b));
    }
}

File: Python Project/calculator.py
from operations import add, subtract, multiply, divide
def main():
    print("Welcome to the Python Calculator!")
    a = 10
    b = 5
    print("Add:", add(a, b))
    print("Subtract:", subtract(a, b))
    print("Multiply:", multiply(a, b))
    print("Divide:", divide(a, b))

if __name__ == "__main__":
    main()
